Ginger/Ginger/SolutionWindows/AddApplicationPage.xaml.cs

```diff
@@ -33,13 +39,16 @@ public partial class AddApplicationPage : Page
     {
         GenericWindow _pageGenericWin = null;
         Solution mSolution;
-
-        public AddApplicationPage(Solution Solution)
+        bool isSolutionNew = false;
+        public AddApplicationPage(Solution Solution , bool isSolutionNew)
```

Comment:
Change the variable name or add two constructors

Reply:
changed the variable name